### ISSUE TITLE
Not working on W8.1 x64

### ISSUE THREAD

**Issue by Nothing4You** [sleighsoft/VLClive#8]
Hi there,

I installed the plugin as suggested in the readme.
After that I started VLC, opened the debug console and when I try to open it I get the following:
```
[00000000025a6c80] main tls client error: TLS client plugin not available
[00000000004d38e0] main generic error: no suitable access module for `https://ra
w.githubusercontent.com/sleighsoft/VLClive/master/src/VLClive.lua'
[00000000004d38e0] lua generic error: Could not activate extension!
```
```
OS: Windows 8.1 x64
VLC: Version 2.1.5 Rincewind x64
```

**Comment by sleighsoft** [comment]
Update vlc to 2.2.1 that should fix it.
GnuTLS is not implemented in 2.1.5

**Comment by Nothing4You** [comment]
That's what you get when you use the builtin updater / update check...
I tried to update earlier already and it told me I would already be running the latest version...
Thanks
